### PR TITLE
feat: visualize cluster scaling metrics

### PR DESCRIPTION
**Pull request by lenaschoenburg**
This adds a new row with basic visualizations for new metrics added in https://github.com/camunda/zeebe/issues/15172.

This also adds our first _Annotation Query_ that automatically marks time regions with a topology change in progress. This makes it easier to correlate other metrics, for example when resource consumption suddenly increases because a broker now joined a new partition.

image

Demo: https://grafana.dev.zeebe.io/d/zeebe-dashboard-cluster/zeebe-with-cluster-row

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -18841,6 +18852,407 @@
...
+              "exemplar": false,
+              "expr": "sum by (pod, operation) (zeebe_cluster_changes_operation_attempts_total{outcome=\"applied\", namespace=~\"$namespace\", pod=~\"$pod\"})",
+              "format": "table",
+              "instant": true,
+              "legendFormat": "{{operation}} on {{pod}}",
+              "range": false,
+              "refId": "A"
+            }
+          ],
+          "title": "Applied Operations",
```
Comment (deepthidevaki) [inline monitor/grafana/zeebe.json:19250]:
```suggestion
          "title": "Number of Applied Operations",
```

**Review by deepthidevaki — APPROVED** [review]
:rocket: Nice. I like the annotation stuff. Is the draft dashboard available in grafana.dev?

:wrench: Could you adjust the histogram buckets for the operation duration? 10s limit is probably too low for the real use cases.

**Comment by oleschoenburg** [comment]
@deepthidevaki I've added the dashboard here: https://grafana.dev.zeebe.io/d/zeebe-dashboard-cluster/zeebe-with-cluster-row

Good point with the histogram, I'll adjust it here.

**Comment by oleschoenburg** [comment]
bors merge

**Pull request merged**